drivers/mipi_dsi/dsi_stm32.c

```diff
@@ -0,0 +1,500 @@
+/*
+ * Copyright (c) 2023 bytes at work AG
+ * Copyright (c) 2020 Teslabs Engineering S.L.
+ * based on dsi_mcux.c
+ * Copyright (c) 2022, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT st_stm32_mipi_dsi
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/mipi_dsi.h>
+#include <zephyr/drivers/reset.h>
+#include <zephyr/logging/log.h>
+
+#include <stm32_ll_rcc.h>
```

Comment:
I don't think it is required anymore. Can you have a check ?

Reply:
Should not be necessary, thanks.